boxoffice/static/js/models/index.js

```diff
@@ -1,5 +1,6 @@
 import {fetch} from './util.js';
 
 export const IndexModel = {
-  fetch: fetch
+  fetch: fetch,
+  url_root: '/admin'
```

Comment:
I'd like this better if `url_root` is a parameter passed in from the HTML template during init, using `request.url_root`.